bootstrap/templates/new/mix.exs

```diff
@@ -59,7 +59,8 @@ defmodule <%= app_module %>.Mixfile do
   # Specify target specific dependencies
   def deps("host"), do: []
   def deps(target) do
-    [{:"nerves_system_#{target}", ">= 0.0.0"}]
+    [{:nerves_runtime, "~> <%= runtime_vsn %>"},
+     {:"nerves_system_#{target}", ">= 0.0.0", runtime: false}]
```

Comment:
The system repositories are synced on version now, but they could diverge. specifying a version in this case would blanket it across all system repos.